scripts/wikipedia.js

```diff
@@ -89,7 +99,21 @@ function attachTooltip (anchor, tooltip) {
     },200);
   })
 }
-function createTooltipWindow (metadata, id) {
+function createDonateToolTip (isbn){
+  var text_elements = $('<div>').attr({'class':'text_elements' }).append(
+			$('<p>').append($('<strong>').text("Please donate $50 and we will try to purchase and digitize the book for you.")),
+		)
+
+	var details = $('<div>').attr({'class':'bottom_details text-muted'}).append(
+		$('<p>').text("Or if you have a copy of this book please mail it to: "),
+    $('<p>').text('300 Funston, San Francisco, CA 94118'),
+    $('<p>').text('so we can digitize it.')
+	)
+
+	return $('<a>').append(text_elements, details).attr({'class': 'popup_box', 'href': 'https://www.archive.org/donate?isbn=' + isbn})[0].outerHTML
```

Comment:
indentation seems broken, do you use `eslint` to check the styling of your code before submitting?
Thank you!